package.json

```diff
@@ -1360,7 +1358,16 @@
             "request": "attach",
             "processId": "${command:pickProcess}"
           }
-        ]
+        ],
+        "windows": {
```

Comment:
I'm not sure why these got added. Presumably they can be removed?

Reply:
Remove these. Otherwise, they'll break the debugger.